Specs/Number/Japanese/NumberModel.json

```diff
@@ -1,10 +1,12 @@
 [
   {
     "Input": "西九条",
+    "Comment": "PendingValidation",
```

Comment:
This is actually a good reminder that the specs should also show negative cases. Cases where we want to make sure entities are not erroneously extracted from certain input text. Please take this into account when revising/extending coverage.